src/plone/restapi/tests/test_serializer_converters.py

```diff
@@ -121,11 +127,11 @@ def test_dict(self):
                                            'baz': 3}))
         self.assertEquals('{"foo": "bar"}',
                           json.dumps(json_compatible({'foo': 'bar'})))
-        self.assertIsInstance(json_compatible({'foo': 'bar'}.keys()[0]),
-                              unicode,
+        self.assertIsInstance(json_compatible(list({'foo': 'bar'}.keys())[0]),
```

Comment:
this should be written: `self.assertIsInstance(json_compatible(list({'foo': 'bar'})[0]),`